lib/dash/mpd_utils.js

```diff
@@ -165,9 +164,10 @@ shaka.dash.MpdUtils = class {
 
       let repeat = r || 0;
       if (repeat < 0) {
-        if (next) {
-          const nextStartTime =
-              XmlUtils.parseAttr(next, 't', XmlUtils.parseNonNegativeInt);
+        if (i + 1 < timePoints.length) {
```

Comment:
Same here it's probably better to have
```
const timePoint = timePoints[i];
const next = timePoints[i + 1];
```
and keep the code with `next` as before

Reply:
Done!